scripts/run-integ-tests.ps1

```diff
@@ -37,15 +39,8 @@ if (-Not ($dockerImages -like "*$BaseImageName*")) {
 }
 Invoke-Expression "docker tag $BaseImageName amazon-ecs-ftest-windows-base:make"
 
-$imgDigest = Invoke-Expression "docker image inspect --format `"{{.RepoDigests}}`" $BaseImageName"
-# The template returns an array so its [$BaseImageDigest]. Remove the '[' and ']'
-$imgDigest = $imgDigest.SubString(1, $imgDigest.Length - 2)
-
-$ProgramFiles="C:\Program Files\Amazon\ECS"
-$env:BASE_IMAGE_NAME=$BaseImageName
-$env:BASE_IMAGE_NAME_WITH_DIGEST=$imgDigest
-
```

Comment:
Added it back in [4e02991](https://github.com/aws/amazon-ecs-agent/pull/4304/commits/4e02991541dc094511ad6cd14c402a3523cfc194) and [725e2b7](https://github.com/aws/amazon-ecs-agent/pull/4304/commits/725e2b785003dc099eea5e25cc9fa21b9e10c9f3). I initially removed the test because it was failing, and I thought that was because it was related to image pulls. A closer inspection revealed the failures were caused by missing state change assertions.

Reply:
It was unused after I removed the Windows tests that used the env var.